internal/kubeconfig/handler_test.go

```diff
@@ -26,6 +25,7 @@ const (
 	instanceID        = "93241a34-8ab5-4f10-978e-eaa6f8ad551c"
 	operationID       = "306f2406-e972-4fae-8edd-50fc50e56817"
 	instanceRuntimeID = "e04813ba-244a-4150-8670-506c37959388"
+	ownClusterPlanID  = "03e3cb66-a4c6-4c6a-b4b0-5d42224debea"
```

Comment:
This plan id is already available by using broker.OwnClusterPlanID.

Reply:
If use broker.OwnClusterPlanID we create an import cycle.